README.md

```diff
@@ -639,6 +640,10 @@ Table of Contents
   * [whimsical.com](https://whimsical.com/) - Collaborative flowcharts, wireframes, sticky notes and mind maps. Create up to 4 free boards.
   * [lensdump.com](https://lensdump.com/) - Free cloud image hosting
 
+## Cookie Consent
+
+  * [metomic.io](https://metomic.io) - Cookie widget which doesn't look bad
```

Comment:
Thanks for the add - unfortunately I'd say I do not want a single entry catagory, we'd need to add it to an existing catagory or find other similar things to make a new one.